instrumentation/logback/logback-appender-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/appender/v1_0/LogbackSingletons.java

```diff
@@ -24,14 +24,20 @@ public final class LogbackSingletons {
     boolean captureCodeAttributes =
         config.getBoolean(
             "otel.instrumentation.logback-appender.experimental.capture-code-attributes", false);
+    boolean captureMarkerAttributes =
+        config.getBoolean(
+            "otel.instrumentation.logback-appender.experimental.capture-marker-attributes", false);
```

Comment:
since only one attribute

```suggestion
            "otel.instrumentation.logback-appender.experimental.capture-marker-attribute", false);
```

Reply:
Done